src/tests/ut/warehouse/warehouse_test.cpp

```diff
@@ -8,10 +8,13 @@
 struct WarehouseTest : ::testing::Test
 {
     Warehouse testWarehouse;
-    std::shared_ptr<Product> anaesthetic = std::make_shared<Medicine>(
-        std::string{"anaesthetic"}, 45, 100, 10122025, std::pair{-10, 0}, std::vector<std::string>{{"chemicals"}});
+    std::chrono::year_month_day expirationDate{static_cast<std::chrono::year>(2030), std::chrono::January,
+                                               static_cast<std::chrono::day>(31)};
+    std::shared_ptr<Product> anaesthetic =
+        std::make_shared<Medicine>(std::string{"anaesthetic"}, 45, 100, expirationDate, std::pair{-10, 0},
+                                   std::vector<std::string>{{"chemicals"}});
     std::shared_ptr<Product> painkiller = std::make_shared<Medicine>(
-        std::string{"painkiller"}, 20, 100, 05062026, std::pair{0, 20}, std::vector<std::string>{"ibuprofenum"});
+        std::string{"painkiller"}, 20, 100, expirationDate, std::pair{0, 20}, std::vector<std::string>{"ibuprofenum"});
```

Comment:
is expirationDate a function? if not, please rename to to snake_case
same in 10 line

Reply:
Done